### ISSUE TITLE
Distinguish old payments from payments that never existed for debugging

### ISSUE THREAD

**Issue by rowgraus** [Agoric/agoric-sdk#3755]
Currently, when payments are used up, they are deleted from the WeakMap. This causes the error that results from reusing a used-up payment to be indistinguishable from something that is not a payment for the brand

**Comment by simpletrontdip** [comment]
Yeah, I have seen some orphaned payments in wallet UI while trying out the demo, call `getAmountOf` shows that they are not in the ledger. These used-up payments keep showing in the wallet UI without being recognized and discarded 👯
